web/src/transmission.js

```diff
@@ -622,7 +622,7 @@ export class Transmission extends EventTarget {
   static _isValidURL(string) {
     try {
       const url = new URL(string);
-      return url ? true : false;
+      return Boolean(url);
```

Comment:
Remove
```Javascript
 return Boolean(url);
```

and just add
```Javascript
return true;
```
at the end of the function.

Reply:
Already suggested in a previous review, but eslint warns `no-new`.

Reply:
Good luck with that then. I'm glad no linter tells me how to write code.

Does it also whine when you just return true within the try-catch?

Reply:
> Good luck with that then. I'm glad no linter tells me how to write code.

Not super productive

> Already suggested in a previous review, but eslint warns `no-new`.

eslint gives `no-new` for `return true;`? That's surprising to me but :man_shrugging:

Reply:
> eslint gives `no-new` for `return true;`? That's surprising to me but 🤷‍♂️

The line that gives `no-new` the previous line instead:

```js
new URL(string); // no-new
return true;
```

This won't work either:

```js
const url = new URL(string); // no-unused-vars
return true;
```

Reply:
> > Good luck with that then. I'm glad no linter tells me how to write code.
> 
> Not a super productive comment.

Maybe not. But ... 
> > Already suggested in a previous review, but eslint warns `no-new`.
> 
> eslint gives `no-new` for `return true;`? That's surprising to me but 🤷‍♂️

Surprise! Change the linter setting then? You can't or you won't because you don't know the impact it might have?